README.md

```diff
@@ -504,16 +504,10 @@ Install HLS along with the following emacs packages:
 
 Make sure to follow the instructions in the README of each of these packages.
 
-Install [use-package](https://github.com/jwiegley/use-package), and add the following to your .emacs
-``` emacs-lisp
-(use-package lsp-haskell
- :ensure t
- :config
- (setq lsp-haskell-process-path-hie "haskell-language-server-wrapper")
- ;; Comment/uncomment this line to see interactions between lsp client/server.
- ;;(setq lsp-log-io t)
-)
-```
+The default `lsp-haskell-server-path` is set to `haskell-language-server-wrapper`. In 
+case you would like your editor to use a specific version of the `hls` server, then this
+variable can be updated. Information on other configurations can be found at 
+[lsp-haskell](https://github.com/emacs-lsp/lsp-haskell)
```

Comment:
I think `doom-emacs` settings should also be fixed accordingly.

Reply:
Doom emacs currently still uses a version of `lsp-haskell` that doesn't have this change. There is a PR to update it (hlissner/doom-emacs#3976), but it might be a bit until it's merged, there's a bit of a backlog at the moment.

Reply:
Hm, OK, then update that after the PR.